tests/README.md

```diff
@@ -76,9 +76,13 @@ Each runtime testcase consists of multiple directives. In no particular order:
 * `TIMEOUT`: The timeout for the testcase (in seconds). This field is required.
 * `BEFORE`: Run the command in a shell before running bpftrace. The command
   will be terminated after testcase is over. Can be used multiple times,
-  commands will run in parallel.
+  commands will run in parallel. `BEFORE` commands are expected to run while
```

Comment:
nit: how about just changing the second sentence to "The command will run while bpftrace is running and be terminated after the test case finishes."